arwen/host/arwen.go

```diff
@@ -28,6 +28,7 @@ var MaximumWasmerInstanceCount = uint64(10)
 var _ arwen.VMHost = (*vmHost)(nil)
 
 const executionTimeout = time.Second
+const signalErrorName = "signalError"
```

Comment:
The logged errors may not have been signalled by the SC (e.g. `OutOfGas`). Consider the rename:
```suggestion
const errorLogIdentifier = "errorLog"
```